REGRESSION(262632@main?): [ iOS ] 2X imported/w3c/web-platform-tests/screen-orientation (layout-tests) are consistent text failures

marcoscaceres opened the pull request:
#### 40235be133be2b67b2eafcbc0dc444d139c66322
<pre>
REGRESSION(262632@main?): [ iOS ] 2X imported/w3c/web-platform-tests/screen-orientation (layout-tests) are consistent text failures
<a href="https://bugs.webkit.org/show_bug.cgi?id=255631">https://bugs.webkit.org/show_bug.cgi?id=255631</a>
rdar://108231120

Reviewed by Chris Dumez.

The utils file didn&apos;t get updated, causing some tests to stall.
Additionally, some of the tests where not pressing on the correct (iframe) contexts.

* LayoutTests/imported/w3c/web-platform-tests/screen-orientation/nested-documents-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/screen-orientation/nested-documents.html:
* LayoutTests/imported/w3c/web-platform-tests/screen-orientation/onchange-event-subframe.html:
* LayoutTests/imported/w3c/web-platform-tests/screen-orientation/resources/orientation-utils.js:
* LayoutTests/imported/w3c/web-platform-tests/screen-orientation/unlock.html:
* LayoutTests/platform/ios/TestExpectations:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/screen-orientation/nested-documents-expected.txt:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/screen-orientation/onchange-event-subframe-expected.txt:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/screen-orientation/unlock-expected.txt:

Canonical link: <a href="https://commits.webkit.org/263224@main">https://commits.webkit.org/263224@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/4ce2fb2ef50ef0eee29e9dd596e43a56c857b162

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/3927 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/4019 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/4130 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/5365 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/4185 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/4104 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/4007 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/3998 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/3975 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/4166 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/3540 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/5217 "Built successfully") | 
| | [⏳ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/iOS-16-Simulator-WPT-WK2-Tests-EWS "Waiting to run tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/3515 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/5556 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/3489 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/3575 "Passed tests") | [❌ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/4987 "12 api tests failed or timed out") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/3979 "Built successfully") | [❌ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/3211 "1 failures") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/3498 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/3515 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/972 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/3547 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/3778 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

cdumez reviewed (changes requested):
You need to drop those tests from LayoutTests/platform/ios/TestExpectations now that you fixed them

cdumez commented on LayoutTests/platform/ios/TestExpectations:
```diff
@@ -4411,9 +4411,3 @@ imported/w3c/web-platform-tests/webaudio/the-audio-api/the-audioparam-interface/
-imported/w3c/web-platform-tests/screen-orientation/nested-documents.html [ Pass Failure ]
-imported/w3c/web-platform-tests/screen-orientation/unlock.html [ Pass Failure ]
-
-webkit.org/b/255704 [ Debug ] editing/inserting/insert-img-uneditable-canonical-position-crash.html [ Skip ]
```
Really?

marcoscaceres replied:
Oh! too quick. Bad rebased. All good now.

cdumez reviewed (approved):
LGTM

marcoscaceres committed:
REGRESSION(262632@main?): [ iOS ] 2X imported/w3c/web-platform-tests/screen-orientation (layout-tests) are consistent text failures
https://bugs.webkit.org/show_bug.cgi?id=255631
rdar://108231120

Reviewed by Chris Dumez.

The utils file didn't get updated, causing some tests to stall.
Additionally, some of the tests where not pressing on the correct (iframe) contexts.

* LayoutTests/imported/w3c/web-platform-tests/screen-orientation/nested-documents-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/screen-orientation/nested-documents.html:
* LayoutTests/imported/w3c/web-platform-tests/screen-orientation/onchange-event-subframe.html:
* LayoutTests/imported/w3c/web-platform-tests/screen-orientation/resources/orientation-utils.js:
* LayoutTests/imported/w3c/web-platform-tests/screen-orientation/unlock.html:
* LayoutTests/platform/ios/TestExpectations:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/screen-orientation/nested-documents-expected.txt:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/screen-orientation/onchange-event-subframe-expected.txt:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/screen-orientation/unlock-expected.txt:

Canonical link: https://commits.webkit.org/263224@main
